TestCases/serial_regression.py

```diff
@@ -496,6 +496,17 @@ def main():
     harmonic_balance.tol       = 0.00001
     test_list.append(harmonic_balance)
 
+    # Turbulent pitching NACA 64a010 airfoil
+    harmonic_balance           = TestCase('turbulent_hb')
+    harmonic_balance.cfg_dir   = "harmonic_balance/turbulent_hb"
+    harmonic_balance.cfg_file  = "davis.cfg"
+    harmonic_balance.test_iter = 25
+    harmonic_balance.test_vals = [-1.784954,-5.471762, 0.007973, 0.077757] #last 4 columns
+    harmonic_balance.su2_exec  = "SU2_CFD"
+    harmonic_balance.timeout   = 1600
+    harmonic_balance.tol       = 0.00001
+    test_list.append(turbulent_hb)
```

Comment:
Jason, 
I see two issues here:
1. please add the mesh file in the corresponding TestCases repository https://github.com/su2code/TestCases . At the moment your input mesh is not given.
2. you named your testcase *turbulent_hb* but here I still see the old name *harmonic_balance*. That's why I would expect an error at line 508. Try to change the names accordingly (e.g.  `harmonic_balance           = TestCase('turbulent_hb')` -> `turbulent_hb = TestCase('turbulent_hb')` and so on).

Maybe we could choose a different name to include the use of preconditioning; something like *hb_rans_preconditioning* ?